Vsts-Extension/Set-AzureAdApplication/Set-AzureAdApplicationV2/task.json

```diff
@@ -222,6 +231,10 @@
         {
             "name": "ServicePrincipalObjectId",
             "description": "The ObjectId of the Service Principal connected to the Azure AD Application"
+        },
+        {
+            "name": "Secret.MyDescription",
+            "description": "The secret generated for the Azure AD Application. If multiple secrets are requested, an output variable for each secret will be available (convention: .Secret.[Description] where [Description] is equal to the Description provided in the 'Secrets' input field)"
```

Comment:
Sorry but can you remove the first . before Secret by ".Secret.[Description]" because it's unclear in my opinion.